disco/cli/upgrade_cost_analysis.py

```diff
@@ -356,20 +351,31 @@ def aggregate_results(output, fmt, verbose):
 
 
 def _aggregate_results(output, log_file, job_names, fmt):
-    upgrade_summary_table = []
-    upgrade_costs_table = []
     jobs_output_dir = output / JOBS_OUTPUT_DIR
     output_json = {
+        "results": [],
+        "costs_per_equipment": [],
         "violation_summary": [],
-        "upgrade_costs": [],
+        "equipment": [],
         "outputs": {"log_file": str(log_file), "jobs": []},
     }
-
+    empty_tables = {"results": [], 'costs_per_equipment': [], 'violation_summary': [], 'equipment': []}
     for name in job_names:
         job_path = jobs_output_dir / name
         job_info = JobInfo(name)
-        summary_table = get_upgrade_summary_table(job_path, job_info)
-        costs_table = get_total_upgrade_costs_table(job_path, job_info)
+        job_keyword = "name"
+        job_name = getattr(job_info, job_keyword)
+        overall_output_summary_file = job_path / "overall_output_summary.json"
+        if not overall_output_summary_file.exists():
+           tables = empty_tables
+        try:
+            with open(overall_output_summary_file) as json_file:
+                data = json.load(json_file)
+        except pd.errors.EmptyDataError:
```

Comment:
This shouldn't be possible...you're doing a json.load. If there is a parsing error, won't it be fatal?